releases/release-1.31/release-notes/maps/pr-124361-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 124361
+releasenote:
+  text: 'kubeadm: don''t mount /etc/pki in kube-apisever and kube-controller-manager
+    pods as an additional Linux system CA location. Mount /etc/pki/ca-trust and /etc/pki/tls/certs
+    instead. /etc/ca-certificate, /usr/share/ca-certificates, /usr/local/share/ca-certificates
+    and /etc/ssl/certs continue to be mounted.'
```

```suggestion
  text: 'kubeadm: Removed support for mounting /etc/pki as an additional Linux system CA location in kube-apisever and kube-controller-manager pods. Instead, it shifted to supporting the mounting of /etc/pki/ca-trust and /etc/pki/tls/certs. The locations /etc/ca-certificate, /usr/share/ca-certificates, /usr/local/share/ca-certificates, and /etc/ssl/certs continued to be supported.'.'
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653209371